app/assets/javascripts/application/notifications.js

```diff
@@ -0,0 +1,8 @@
+Utils.fireWhenReady(['notifications#index', 'notifications#notifications'], function(e) {
+  $('.approve-sign-slide').on('click', function(event) {
+    var message = $(this).parent().siblings('.input-field').children('input').val();
+    $(this).attr('data-message', message);
+    $.post($(this).attr('href') + "?message=" + message );
+    return false;
+  });
+});
```

Comment:
This was some of that Signage js that I was bitching about. I really didn't understand why stuff in the /application folder works only for non-admin users and stuff in the /admin folder works only for admin users. This duplicated js is a result of that being an issue.